test/memspaces/memspace_fixtures.hpp

```diff
@@ -65,9 +65,10 @@ struct memspaceGetTest : ::numaNodesTest,
         }
 
         auto [isQuerySupported, memspaceGet] = this->GetParam();
+        isQuerySupported(nodeIds.front());
 
-        if (!isQuerySupported(nodeIds.front())) {
-            GTEST_SKIP();
+        if (IS_SKIPPED_OR_FAILED()) {
+            return;
```

Comment:
hmm... so what happens when we return in the SetUp function? will the test skip or what?

Reply:
The test has been marked as skipped in isQuerySupported() using GTEST_SKIP(), repeating GTEST_SKIP() would only duplicate the skip message in the output. I'll add a comment in the code